go.mod

```diff
@@ -1,24 +1,23 @@
 module github.com/coveooss/tgf
 
-go 1.12
-
-replace github.com/gruntwork-io/terragrunt => github.com/coveooss/terragrunt v1.4.0
+go 1.13
```

Comment:
Why not upgrade to go 1.14?

Reply:
In fact, I don't remember changing this line. But the go version specified here does not specify the version used to compile the code, but the minimum requirement if this module is imported from elsewhere. If I put 1.14 here, I will have to update all projects including this project (which is none in that specific case 😀 )